storage/api/Storage.sln

```diff
@@ -54,6 +56,18 @@ Global
 		{22A2AE8E-6635-496E-A52A-EAA63361EEAA}.Release|x64.Build.0 = Release|Any CPU
 		{22A2AE8E-6635-496E-A52A-EAA63361EEAA}.Release|x86.ActiveCfg = Release|Any CPU
 		{22A2AE8E-6635-496E-A52A-EAA63361EEAA}.Release|x86.Build.0 = Release|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Debug|x64.Build.0 = Debug|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Debug|x86.Build.0 = Debug|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Release|Any CPU.Build.0 = Release|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Release|x64.ActiveCfg = Release|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Release|x64.Build.0 = Release|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Release|x86.ActiveCfg = Release|Any CPU
+		{C0736C40-22CC-467A-8020-A965AD16AF9B}.Release|x86.Build.0 = Release|Any CPU

```

Comment:
This is about to add `testutil` project in current solution as Visual Studio was complaining about `testutil` project.